src/utils/error-reporting.ts

```diff
@@ -0,0 +1,25 @@
+import * as Sentry from '@sentry/react';
+
+let initialised = false;
+export const initSentry = () => {
+  if (initialised) return;
```

Comment:
Previously there was also condition checking if `process.env.REACT_APP_SENTRY_DSN` is defined. Maybe it can be added here too.